docs/how-to/installation/install-on-kubernetes.txt

```diff
@@ -0,0 +1,166 @@
+.. _install-on-kubernetes:
+
+Install Determined on Kubernetes
+================================
+
+This document describes how to install Determined on `Kubernetes <https://kubernetes.io/>`_. The installation
+is performed using the :download:`Determined Helm Chart </helm/determined-helm-chart.tgz>`.
+When installing using the Helm chart, all configurations will be done within
+`values.yaml <https://github.com/determined-ai/determined/blob/master/helm/charts/determined/values.yaml>`_ and
+`Chart.yaml <https://github.com/determined-ai/determined/blob/master/helm/charts/determined/Chart.yaml>`_.
+
+
+Prerequisites
+-------------
+
+Before installing Determined on a Kubernetes cluster, please ensure that the
+following prerequisites are satisfied:
+
+* The Kubernetes cluster should be running Kubernetes >= 1.15, with GPU support enabled. :ref:`Instructions for setting up a GKE cluster <setup-gke-cluster>`.
+* You should have access to the cluster via `kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_.
+* `Helm <https://helm.sh/docs/helm/helm_install/>`_ should be installed.
+* You should have a local copy of the:download:`Determined Helm Chart </helm/determined-helm-chart.tgz>`.
+
+
+What Gets Installed
+-------------------
+
+When the Determined Helm chart is installed, the following entities will be created:
+
+#. Deployment of the Determined master.
+#. ConfigMap containing configurations for the Determined master.
+#. LoadBalancer service to make the Determined master accessible.
+#. ServiceAcccount which will be used by the Determined master.
+#. Deployment of a Postgres database. Later in this guide, we demonstrate how an external database can be used instead.
+#. PersistentVolumeClaim for the Postgres database. Omitted if using an external database.
+#. Service to allow the Determined master to communicate with the Postgres database. Omitted if using an external database.
+
+
+.. _configure-determined-kubernetes-version:
+
+Configuring Which Version of Determined to Install
+--------------------------------------------------
+
+To configure which version of Determined users need to modify ``appVersion`` in ``Chart.yaml``. Users can specify
+a release version (e.g., ``0.13.0``) or specify any commit hash from the `upstream Determined repo <https://github.com/determined-ai/determined>`_
+(e.g., ``b13461ed06f2fad339e179af8028d4575db71a81``). Users are encouraged to use a released version.
+
+.. note::
+
+    If using a non-release branch of the Determined repository, ``appVersion`` is going to be set to ``X.Y.Z.dev0``.
```

Comment:
I removed this bc it seemed like an implementation detail most users shouldn't need to care about.